oar/cli/cmd_drop_bugs.py

```diff
@@ -0,0 +1,38 @@
+import click
+import logging
+from oar.core.worksheet_mgr import WorksheetManager
+from oar.core.advisory_mgr import AdvisoryManager
+from oar.core.config_store import ConfigStore
+from oar.core.const import *
+
+logger = logging.getLogger(__name__)
+
+
+@click.command()
+@click.pass_context
+def drop_bugs(ctx):
+    """
+    Drop bugs from advisories
+    """
+    # get config store from context
+    cs = ctx.obj["cs"]
+    try:
+        # get existing report
+        report = WorksheetManager(cs).get_test_report()
+        # init advisory manager
+        am = AdvisoryManager(cs)
+        # update task status to in progress
+        report.update_task_status(LABEL_TASK_DROP_BUGS, TASK_STATUS_INPROGRESS)
```

Comment:
do we need to update task status to in progress before dropping bugs?

Reply:
yes, I think so.